contracts/contribute/ContributeManager.sol

```diff
@@ -109,13 +123,15 @@ contract ContributeManager is ERC721TokenReceiver {
         address _olas,
         address _stakingFactory,
         address _safeMultisig,
+        address _safeSameAddressMultisig,
```

Comment:
Can we not look this up from the whitelist? Would mean less hardcoding. What if this updates later?

Reply:
No we can't, in ServiceRegistry there's just a map with whitelisted multisigs.

Reply:
We can pass them as function arguments every time, so if they change - no big deal.